README.md

```diff
@@ -1,32 +1,37 @@
 
-Temporal is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way.
+This repo contains a basic [Helm](https://helm.sh) chart that installs Temporal to a Kubernetes cluster. The dependencies that are bundled with this installation solution offer an easy way to **experiment** with the Temporal server.
 
-This repo contains a basic [Helm](https://helm.sh) chart that allows you to install temporal to a kubernetes cluster, and to play with it.
-
-The version of Helm chart is provided for demo purposes and is not intended to be used in production systems.
-
-# Deploying Temporal Service to a Kubernetes Cluster
+# Install Temporal service on a Kubernetes cluster
 
 ## Prerequisites
 
-This sequence assumes that your system is configured to access a kubernetes cluster (e. g. [AWS EKS](https://aws.amazon.com/eks/), or [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)), and that your machine has [AWS CLI V2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html), [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [Helm v3.1.x](https://helm.sh) installed and able to access your cluster.
-
-## Install an Instance of Temporal to Your k8s Cluster
+This sequence assumes that your system is configured to access a kubernetes cluster (e. g. [AWS EKS](https://aws.amazon.com/eks/), [kind](https://kind.sigs.k8s.io/), or [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)), and that your machine has [AWS CLI V2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html), [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [Helm v3.1.x](https://helm.sh) installed and able to access your cluster.
 
 Download Helm dependencies:
 
 ```bash
 ~/temporal-helm$ helm dependencies update
 ```
 
-### Install Temporal
+## Install Temporal Helm chart
+
+Temporal can be configured to run with a variety of different dependencies and the Helm chart installs these by default:
+
+* Cassandra
+* ElasticSearch
+* Kafka (with Zookeeper)
+* Promethueus
+* Grafana
+
+MySQL can be swapped in for Cassandra but is not deployed as part of this Helm chart.
 
-Temporal can be configured to run with a couple of database choices.
+The following sections will walk through installing with everything included on a single node cluster and conclude with a configurtion that lets you bring all of your own dependencies.
 
-#### [Tiny] Batteries Included
+### Install minimal versions of only requried dependencies
```

Comment:
Ya, I like this direction better. But I suggest rephrasing it to:
```suggestion
### Minimal installation with required dependencies only
```

However, not sure if that loses the intended meaning - as this sentence "Install minimal versions of only requried dependencies" could very easily be suggesting that minimal versions is referring to the dependency versions. So just check me on that -